internal/consensus/state.go

```diff
@@ -2273,19 +2273,14 @@ func (cs *State) tryAddVote(vote *types.Vote, peerID p2p.ID) (bool, error) {
 			)
 
 			return added, err
-		} else if errors.Is(err, types.ErrVoteNonDeterministicSignature) {
-			cs.Logger.Info("Vote has non-deterministic signature", "err", err)
```

Comment:
This will become an Error level message, see line 965.

Reply:
This line is covered by `TestByzantineConflictingProposalsWithPartition` test unit.